### PR TITLE
Master yml yti

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mreficent** [comment]
:heart::heart::heart::heart::heart::heart::heart::heart::heart:

keep with this nice work :punch:

**Comment by tivisse** [comment]
@rco-odoo Could you please review the commit https://github.com/odoo/odoo/pull/20890/commits/e70128fba5f430a0b5516bd420801978ac98e13a

It's about the yml engine removal.

Thanks.

**Comment by tivisse** [comment]
Thanks @mreficent  :)

File: addons/auth_oauth/data/auth_oauth_data.xml [hunk 1]
```diff
@@ -29,5 +29,13 @@
             <field name="css_class">fa fa-google-plus-square</field>
             <field name="body">Log in with Google</field>
         </record>
+
+        <!-- Use database uuid as client_id for OpenERP oauth provider -->
+        <function model="auth.oauth.provider" name="write">
+            <value eval="[ref('auth_oauth.provider_openerp')]"/>
```
Comment (rco-odoo) [inline addons/auth_oauth/data/auth_oauth_data.xml]: simpler as
```xml
<value ref="auth_oauth.provider_openerp"/>
```

File: odoo/tests/addons/test_translation_import/tests/__init__.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 
 from . import test_term_count
-
+from . import test_translation_import
```
Comment (rco-odoo) [inline odoo/tests/addons/test_translation_import/tests/__init__.py]: ???

File: addons/auth_oauth/data/auth_oauth_data.xml (hunk 1 above)
Comment (tivisse) [reply addons/auth_oauth/data/auth_oauth_data.xml]: En fait, non ça ne marche pas :)

**Pull request merged**